roles/pulp_common/vars/main.yml

```diff
@@ -7,6 +7,15 @@ pulp_install_plugins_normalized_yml: |-
   {% for key, value in pulp_install_plugins.items() %}
   {{ key.replace('_', '-') }}: {{ value | to_json }}
   {% endfor %}
+# List of plugin to ignore when collecting static files
+pulp_collectstatic_ignore_list: |-
+  {{ pulp_install_plugins | dict2items |
+      selectattr('value.collectstatic', "defined") |
+      selectattr('value.collectstatic', "equalto", False) |
+      map(attribute='key') |
+      map('regex_replace', "-", "_") |
```

Comment:
To understand this correctly: This is the opposite of what `pulp_install_plugins_normalized` provides? All "_" no "-"?

Reply:
Correct.

```
pulp_install_plugins:
 pulp-ansible:
 pulp-container:
```

Translates to:

```
/usr/lib/python3.6/site-packages/pulp_ansible
/usr/lib/python3.6/site-packages/pulp_container
```

Reply:
Why not just use `pulp_install_plugins_normalized`? The defaults is evaluated in sequential order.